examples/aws-docker-build-deploy-dev.yaml

```diff
@@ -0,0 +1,75 @@
+name: <App> - Deployment [Dev]
```

Comment:
it was used CD over Deployment due to the reasons:
- shorter. StrongSD - Deployment [Production] vs StrongSD - CD [Production] 
- CD is both build and deploy, Deployment is deploy only